### PR TITLE
add sql_query support

### PR DESCRIPTION
**Pull request by feng19**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -28,7 +28,8 @@ defmodule ExAliyunOts.Mixfile do
 
   defp deps do
     [
-      {:tablestore_protos, "~> 0.1.4"},
+      #{:tablestore_protos, "~> 0.1.4"},
+      {:tablestore_protos, github: "feng19/tablestore_protos"},
```
Comment (xinz) [inline mix.exs:32]: @feng19 抽空这里也帮忙提供一个更新PR

**Review by xinz — APPROVED** [review]
抱歉才回复，多谢~ : )

File: mix.exs (hunk 1 above)
Comment (feng19) [reply mix.exs:32]: tablestore_protos PR: https://github.com/xinz/tablestore_protos/pull/2

Comment (xinz) [reply mix.exs:32]: Merged and released in 0.1.5

> tablestore_protos PR: [xinz/tablestore_protos#2](https://github.com/xinz/tablestore_protos/pull/2)

**Pull request merged**